packages/cardpay-cli/scheduled-payment/schedule-payment.ts

```diff
@@ -1,51 +1,141 @@
 import { Arguments } from 'yargs';
 
 export default {
-  command: 'schedule-payment <safeAddress> <safeModuleAddress> <gasTokenAddress> <spHash>',
-  describe: ' ',
+  command:
+    'schedule-payment <safeAddress> <moduleAddress> <tokenAddress> <amount> <payeeAddress> <fixedUSDFee> <feePercentage> <executionGas> <maxGasPrice> <gasTokenAddress> <salt> <payAt> <recurringDayOfMonth> <recurringUntil>',
+  describe:
+    "Schedules a one-time or a recurring payment. This creates a record in the crank (hub) and stores the scheduled payment hash in the safe's scheduled payment module contract. The crank is responsible for executing the payment at the scheduled time.",
   builder(yargs: Argv) {
     return yargs
       .positional('safeAddress', {
         type: 'string',
         description: 'The address of the safe that will fund the scheduled payment',
       })
-      .positional('safeModuleAddress', {
+      .positional('moduleAddress', {
         type: 'string',
         description: 'The address of the scheduled payment safe module',
       })
+      .positional('tokenAddress', {
+        type: 'string',
+        description: 'The address of the token to be transferred',
+      })
+      .positional('amount', {
+        type: 'string',
+        description: 'The amount of tokens to be transferred',
+      })
+      .positional('payeeAddress', {
+        type: 'string',
+        description: 'The address of the payee (recipient)',
+      })
+      .positional('fixedUSDFee', {
+        type: 'number',
+        description: 'Fixed USD fee (e.g. 0.25)',
+      })
+      .positional('percentageFee', {
+        type: 'number',
+        description: 'Percentage fee (e.g. 5%, 0.05)',
+      })
+      .positional('executionGas', {
+        type: 'number',
+        description: 'The gas limit to execute scheduled payment',
+      })
+      .positional('maxGasPrice', {
+        type: 'string',
+        description: 'Maximum gas price (in the smallest units of gas token)',
+      })
       .positional('gasTokenAddress', {
         type: 'string',
         description: 'The address of the gas token',
       })
-      .positional('spHash', {
+      .positional('salt', {
         type: 'string',
-        description: 'Keccak hash of the scheduled payment params',
+        description: 'Arbitrary string to make SP unique',
+      })
+      .positional('payAt', {
+        type: 'number',
+        description:
+          'The unix UTC time in seconds. Used for one-time scheduled payments. Should be an empty string when recurringDayOfMonth and recurringUntil are set',
+      })
+      .positional('recurringDayOfMonth', {
+        type: 'number',
+        description:
+          'Day of the month on which the payment will be made recurringly (range: 1-31). Used for recurring scheduled payments. In case the month has less than days than the value provided, the payment will me made on the last day of the month. Should be an empty string when payAt is set.',
```

Comment:
We originally planned to have this param be limited in the 1-28 range but I think @habdelra's comment here makes sense: https://github.com/cardstack/cardstack/pull/3220#discussion_r970995395